packages/hash/api/src/graphql/resolvers/index.ts

```diff
@@ -86,12 +77,12 @@ import { createPersistedComment } from "./knowledge/comment/comment";
 import { persistedBlocks } from "./knowledge/block/block";
 import { getBlockProtocolBlocks } from "./blockprotocol/getBlock";
 import {
-  createPersistedEntity,
-  getPersistedEntity,
-  getAllLatestPersistedEntities,
-  updatePersistedEntity,
+  createEntityWithMetadata,
+  getEntityWithMetadata,
+  getAllLatestEntitiesWithMetadata,
+  updateEntityWithMetadata,
```

Comment:
Are we naming these to avoid conflicts for now? Because ideally these would just be `createEntity` and such, right?

Reply:
Yes, some of them we could get away with calling just `Entity`, but for consistency after deprecating/removing the old ones, we can simply search/replace `WithMetadata`

Reply:
I think that makes sense to not get stuck in refactoring hell. cc @CiaranMn

Reply:
Let's make getting rid of old stuff something we do immediately after merging to main